bin/update_aws_spec.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+import yaml
+
+from localstack.utils.openapi import get_localstack_openapi_spec
+from localstack.version import version as ls_version
+
+import click
+
+@click.command()
+@click.option('--latest', is_flag=True, default=False, help='If enabled, sets the version of the spec to latest.')
```

Comment:
nit: I'm not sure if it's necessary to add a dependency on click just to add an argument, but that's just a very subjective opinion and click is pretty cool :P

Reply:
As you correctly mentioned above, `click` comes transitively with LocalStack so I don't see harm using it :)